### ISSUE TITLE
WOA vs ACCESS-OM2 restarts in global simulation.

### ISSUE THREAD

**Issue by AndyHoggANU** [COSIMA/mom6-panan#36]
We have been thinking that the best initial condition for the panantarctic run might be to revert to WOA. 

As part of the testing for the initial condition, and the dependence of southern ocean upwelling and abyssal overturning, we have run three cases:
* `global-01-v1` - the first attempt at a global 0.1° z* simulation, starting from ACCESS-OM2 spunup state.
* `global-01-v2` - as above, but starting from WOA
* `global-01-v3` - as for v2 but with the HYCOM1 vertical coordinate.

I've now run a quick analysis on the first ~30 years of these runs:
https://github.com/AndyHoggANU/mom6-diagnostics/blob/main/global-01/ComparisonWithPanant.ipynb
In short, we seem to have a problem with the creation of dense water -- it is pretty strong when we start from ACCESS-OM2 but seems to disappear after about 20 years of running from WOA. This problem is even worse in the HYCOM1 coordinate than in z*...

This needs more investigation, but I think the takeaway message is that, for now, we need to investigate more before switching to HYCOM1. And we may want to carefully consider our initial condition.

**Comment by AndyHoggANU** [comment]
PS. Do we have a PanAnt 0.1° case with the WOA restart?

**Comment by adele157** [comment]
Yes there are panan-01 cases starting from yr2 of OM2 and yr2 of ACCESS-OM2-01 (so pretty much WOA). Nearly 20 years of each. They are here if you want to add them to your analysis:

`/scratch/e14/akm157/mom6/archive/panan-01-zstar_OM4_IC_BF/` -> OM4
`/scratch/e14/akm157/mom6/archive/panan-01-zstar-yr2forcing/` -> ACCESS-OM2-01 yr 2

In this database:
`/g/data/v45/akm157/model_data/mom6/panan/panan.db`

**Comment by adele157** [comment]
It's hard to tell what's going on with those noisy overturning time series - can you put some yearly averages on them? The time series don't look too terrible, but I agree the overturning streamfunctions don't look so good right in the south. 

I've been meaning to write some code to include in these evaluations of the cross 1000m isobath DSW transport - should be easy and fast to calculate from the MOM6 output. Will send to you if I get around to this in the short term.

**Comment by AndyHoggANU** [comment]
Would love to incorporate your latest runs but I can't see `e14` ... can you sync them to `ik11`?

**Comment by AndyHoggANU** [comment]
Yeah, agreed that the X-isobath transport would be good here.

**Comment by adele157** [comment]
Yes, sorry, will let you know when they're on ik11.

**Comment by adele157** [comment]
New panan-01 runs starting close to obs and with different boundary forcing are now on ik11. They are `panant-01-zstar-OM4` and `panant-01-zstar-ACCESSyr2`.

**Comment by AndyHoggANU** [comment]
Great, but how did you transfer? There seem to be some permission issues....

**Comment by AndyHoggANU** [comment]
```
du: cannot read directory './panant-01-zstar-ACCESSyr2/output034': Permission denied
du: cannot read directory './panant-01-zstar-ACCESSyr2/output031': Permission denied
du: cannot read directory './panant-01-zstar-ACCESSyr2/output030/manifests': Permission denied
du: cannot read directory './panant-01-zstar-ACCESSyr2/output033': Permission denied
du: cannot read directory './panant-01-zstar-ACCESSyr2/output032': Permission denied
854G	./panant-01-zstar-ACCESSyr2
du: cannot read directory './panant-01-zstar-OM4/output034': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output028': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output022': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output020': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output019': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output021': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output027': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output029': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output031': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output018/manifests': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output026': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output030': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output024': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output023': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output033': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output032': Permission denied
du: cannot read directory './panant-01-zstar-OM4/output025': Permission denied
```

**Comment by adele157** [comment]
Hmm, it looks like the files are owned by ik11 to me.

`[akm157@gadi-login-05 mom6-panan]$ ls -l panant-01-zstar-OM4/output020/
total 52006992`
`-rw-r-----+ 1 akm157 ik11   620474059 Mar 27 12:45 20010101.ice_daily.nc`
`-rw-r-----+ 1 akm157 ik11    26555198 Mar 27 12:45 20010101.ice_month.nc`
`-rw-r-----+ 1 akm157 ik11  6900744257 Mar 27 12:41 20010101.ocean_daily.nc` etc

**Comment by angus-g** [comment]
Some of the `output???` directories aren't group-readable.

**Comment by adele157** [comment]
Weird, I couldn't see that. Better now though?

**Comment by angus-g** [comment]
I'm still not seeing them (`output020` through `output034`) as g+rx or g+rS (for sticky bit):
```
$ ls -ld /g/data/ik11/outputs/mom6-panan/panant-01-zstar-OM4/output020
drwx--S---+ 3 akm157 ik11 12288 Mar 27 12:45 /g/data/ik11/outputs/mom6-panan/panant-01-zstar-OM4/output020
```

**Comment by aekiss** [comment]
Looks like an ACL mask problem
```
$ getfacl /g/data/ik11/outputs/mom6-panan/panant-01-zstar-OM4/output020
getfacl: Removing leading '/' from absolute path names
# file: g/data/ik11/outputs/mom6-panan/panant-01-zstar-OM4/output020
# owner: akm157
# group: ik11
# flags: -s-
user::rwx
group::r-x			#effective:---
group:ik11_w:rwx		#effective:---
mask::---
other::---
default:user::rwx
default:group::r-x
default:group:ik11_w:rwx
default:mask::rwx
default:other::---
```

**Comment by aekiss** [comment]
maybe something like this would fix it? @angus-g what do you reckon?
```
setfacl -R -m m::rwX /g/data/ik11/outputs/mom6-panan/panant-01-zstar-OM4/*
```

**Comment by aekiss** [comment]
@adele157 try the command above

**Comment by adele157** [comment]
Tried that. Good now?
